tests/test_omp_extension_install.py

```
@@ -407,42 +442,140 @@ def main() -> int:
+                if line.strip()
+            ]
+            args_lines = [
+                line
+                for line in fake_args_log.read_text(encoding="utf-8").splitlines()
+                if line.strip()
+            ]
+            for raw_pid in pid_lines:
+                try:
+                    os.kill(int(raw_pid), signal.SIGKILL)
+                except (ProcessLookupError, ValueError):
+                    pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(ProcessLookupError, ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ProcessLookupError, ValueError): ...`

(SIM105)